apps/generator/lib/generator.js

```diff
@@ -135,7 +135,7 @@ class Generator {
         enumerable: true,
         get() {
           if (!self.templateConfig.parameters?.[key]) {
-            throw new Error(`Template parameter "${key}" has not been defined in the package.json file under generator property. Please make sure it's listed there before you use it in your template.`);
+            throw new Error(`Template parameter "${key}" has not been defined in the .ageneratorrc file. Please make sure it's listed there before you use it in your template.`);
```

Comment:
I bet in future more formats will be added, wouldn't it be better to do:

```suggestion
            throw new Error(`Template parameter "${key}" has not been defined in the Generator configuration. Please make sure it's listed there before you use it in your template.`);
```